workspaces/demo-app/setup-tests.js

```diff
@@ -4,4 +4,4 @@ import { createSerializer } from 'jest-emotion'
 import * as emotion from 'emotion'
 
 expect.addSnapshotSerializer(createSerializer(emotion))
-jest.setTimeout(7000)
+jest.setTimeout(9000)
```

Comment:
Did you mean to commit this change? It doesn't seem to belong in this pull request..

Reply:
Locally I needed this, but indeed... this should not have been committed!